docs/README.md

```diff
@@ -209,6 +209,8 @@ information.
 	  Provides a common base for creating and managing Decred network peers.
     * [blockchain](https://github.com/decred/dcrd/tree/master/blockchain) -
 	  Implements Decred block handling and chain selection rules
+    * [blockchain/fullblocktests](https://github.com/btcsuite/decred/tree/master/blockchain/fullblocktests) -
+    Provides a set of block tests for testing the consensus validation rules
```

Comment:
The entries are not consistent in leading spaces which is what the intent of the commit being merged from upstream was.  It needs to be applied to not only this line, but all the others as well.

In particular notice how the lines with the bullet points have leading spaces while the continuation lines are tabs.  They all need to be spaces to be consistent.